examples/index.json

```diff
@@ -2493,6 +2493,29 @@
       }
     ]
   },
+  {
+    "title": "  GraphQL service advanced",
+    "column": 1,
+    "category": "Network libraries",
+    "samples": [
+      {
+        "name": "Dataloader",
+        "url": "graphql-dataloader",
+        "verifyBuild": true,
+        "verifyOutput": false,
+        "disablePlayground": true,
+        "isLearnByExample": false
+      },
+      {
+        "name": "Custom prefetch methods",
+        "url": "custom-prefetch-methods",
+        "verifyBuild": true,
+        "verifyOutput": false,
+        "disablePlayground": true,
+        "isLearnByExample": false
+      }
```

Comment:
Maybe we can move some of the samples from the `GraphQL Service` section to here. @praneesha are there any complications of changing the BBE category?

Reply:
We can move.

Reply:
updated with  https://github.com/ballerina-platform/ballerina-distribution/pull/4941